test/k8sT/Health.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sTest
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/cilium/cilium/test/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+var testName = "K8sHealthTest"
+
+var _ = Describe(testName, func() {
+
+	var kubectl *helpers.Kubectl
+	var logger *logrus.Entry
+	var once sync.Once
+	initialize := func() {
+		logger = log.WithFields(logrus.Fields{"testName": testName})
+		logger.Info("Starting")
+
+		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		path := kubectl.ManifestGet("cilium_ds.yaml")
+		kubectl.Apply(path)
+		_, err := kubectl.WaitforPods(helpers.KubeSystemNamespace, "-l k8s-app=cilium", 600)
+		Expect(err).Should(BeNil())
+	}
+
+	BeforeEach(func() {
+		once.Do(initialize)
+	})
+
+	AfterEach(func() {
+		if CurrentGinkgoTestDescription().Failed {
```

Comment:
@eloycoto on this case, when is the test cleaned up?

Reply:
The `cilium_ds.yaml` is use to be in there for all the kubernetes test. No new deployments, pods are in place. So nothing to cleanup.